content/applications/websites/ecommerce/managing_products/price_management.rst

```diff
@@ -0,0 +1,204 @@
+
+.. note::
+   Pay attention that having the price per unit may be **mandatory** in some countries.
+
+.. seealso::
+   - :doc:`../../../inventory_and_mrp/inventory/management/products/uom`
+
+Price configuration
+===================
+
+Pricelists
+----------
+
+Pricelists are the main tool to manage prices on your ecommerce. They allow you to define
+website-specific prices - different from the price on the product template - based on **country
+groups**, **currency**, **minimum quantity**, **period**, or **variants**. You can create as many
+pricelist as needed, but it is mandatory to have at least one pricelist available on a website. If
+no custom pricelists are added, Odoo defaults to the **Public Pricelist** for all websites.
+
+Configuration
+~~~~~~~~~~~~~
+
+Pricelists can be found under :menuselection:`Website app --> eCommerce --> Pricelists`, but must
+first be activated. For that, head to :menuselection:`Website app --> Configuration --> Settings`
+and scroll-down to the :guilabel:`Shop - Products` section. There, you can find two options:
+
+- :guilabel:`Multiple prices per product`;
+- :guilabel:`Advanced price rules (discounts, formulas)`.
+
+The **first** option allows you to set different prices per customer *segment*, i.e. registered
+customers, gold customers, regular customers, etc. The **second** option allows you to set *price
+change* rules such as **discounts**, **margins**, **roundings**, etc.
+
+Foreign currency
+~~~~~~~~~~~~~~~~
+
+If you are selling in **multiple currencies** and have pricelists in foreign currencies, customers
+can select their coresponding pricelist anywhere on the :guilabel:`Shop` page from the drop-down
+menu next to the **search bar**.
+
+.. image:: price_management/price-pricelists.png
+   :align: center
+   :alt: Pricelists selection
+
+.. seealso::
+   - :doc:`../../../sales/sales/products_prices/prices/pricing`
+   - :doc:`../../../sales/sales/products_prices/prices/currencies`
+
+Discount and loyalty programs
+=============================
+
+It is possible to create promotion and loyalty programs, which offer different options from
+pricelists. To do so, go to :menuselection:`Website app --> eCommerce --> Discount & Loyalty`, and
+:guilabel:`Create` a program from scratch or by using the templates. You can also modify an existing
+program by clicking on it.
+
+.. image:: price_management/price-discount-loyalty.png
+   :align: center
+   :alt: Discount and loyal program templates
+
+.. _pricing_management/program-types:
+
+The different program types are:
+
+- :guilabel:`Coupons`: Generate and share single-use, coupon codes that grant access to rewards;
+- :guilabel:`Next Order Coupons`: Generate and share single-use, coupons that grant access to
+  rewards on the next order;
+- :guilabel:`Promotions`: Set conditional rules when ordering, which grant access to reward for the
+  customers;
+- :guilabel:`Discount Code`: Set codes which when entered upon checkout grants customers a discount;
```

Comment:
```suggestion
Discount and loyalty programs
=======================

It is possible to create promotion and loyalty programs, which offer different options from
pricelists. To do so, go to :menuselection:`Website app --> eCommerce --> Discount & Loyalty`, and
:guilabel:`Create` a program from scratch or by using the templates. You can also modify an existing
program by clicking on it.

.. image:: price_management/price-discount-loyalty.png
   :align: center
   :alt: Discount and loyal program templates

.. _pricing_management/program-types:

The different program types are:

- :guilabel:`Coupons`: Generate and share single-use, coupon codes that grant access to rewards;
- :guilabel:`Next Order Coupons`: Generate and share single-use coupons that grant access to
  rewards on the next order;
- :guilabel:`Promotions`: Set conditional rules when ordering, which grant access to rewards for the
  customers;
- :guilabel:`Discount Code`: Set codes which, when entered upon checkout, grants customers a discount;
```

Reply:
Same issue with title length here too. I did change the spelling and commas :)